hack/helm_vars/wire-server/values.yaml.gotmpl

```
@@ -400,6 +405,7 @@ galley:
     rabbitmq:
       username: {{ .Values.rabbitmqUsername }}
       password: {{ .Values.rabbitmqPassword }}
+    pgPassword: "posty-the-gres"
```

```suggestion
    pgPassword: posty-the-gres
```

I'd say `posty-the-gres` and `"posty-the-gres"` are equivalent.